examples/models/llama/tests/test_export_llama_lib.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+
+from executorch.devtools.backend_debug import get_delegation_info
+from executorch.examples.models.llama.export_llama_lib import _export_llama, build_args_parser
+
+
+# Ops expected to be found in the default exported llama_transformer. Obtained through
+# print_delegation_info from the backend_debug module, which is displayed with
+# export_llama under --verbose.
+BASE_EXPECTED_OPS = {
```

Comment:
I think this might be a little risky to test. If the IR changes which is a common possibility this count will change and we'll have to keep fixing this.